src/routes/Tool/Tool.tsx

```diff
@@ -108,50 +110,64 @@ const MyOwnData = () => {
     [navigate, tabs],
   );
 
+  // 100vh - header height - padding
+  const virtualAssistantClassNames: string = toolStore.virtualAssistantOpen ? 'h-[calc(100vh-96px-40px)]' : '';
```

Comment:
Little hacky to get the height for the virtualassistant.

If **ErrorNote** is shown, that height is not taken into calculation because it felt too much logic. And ErrorNote should be rare case or it needs immediate action from the user, so it should be okay for it to push content.